part-1

AlexanderVVasilenko opened the pull request.

AlexanderVVasilenko committed:
done

AlexanderVVasilenko committed:
done

AlexanderVVasilenko committed:
added structure

AlexanderVVasilenko committed:
done

levovit commented on main.py:
```diff
@@ -0,0 +1,82 @@
+    lap_time: str
+
+
+# Load abbreviations from the file
```
remove useless comments

levovit reviewed (changes requested):
I think it's better to move end.log and start.log  and abbreviations to different folders. Like...input_data/

AlexanderVVasilenko committed:
report.py: some changes

AlexanderVVasilenko committed:
Merge pull request #3 from AlexanderVVasilenko/part-2

part-2

AlexanderVVasilenko committed:
report.py:
print_remaining_racers(): added
print_top_15_racers(): added
print_report(): changed with using previous 2 functions

README.md: changed cli command for installing package

AlexanderVVasilenko committed:
main(): renamed to initialize_data()

AlexanderVVasilenko committed:
Merge pull request #5 from AlexanderVVasilenko/part-2

Part 2

AlexanderVVasilenko committed:
Merge pull request #2 from AlexanderVVasilenko/part-1-cli

part-1-cli
